src/main/java/dev/j3fftw/litexpansion/items/PortableCharger.java

```diff
@@ -0,0 +1,225 @@
+package dev.j3fftw.litexpansion.items;
+
+import dev.j3fftw.litexpansion.Items;
+import dev.j3fftw.litexpansion.LiteXpansion;
+import dev.j3fftw.litexpansion.utils.Utils;
+import io.github.thebusybiscuit.slimefun4.core.attributes.Rechargeable;
+import io.github.thebusybiscuit.slimefun4.core.handlers.ItemUseHandler;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunItems;
+import io.github.thebusybiscuit.slimefun4.implementation.items.SimpleSlimefunItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.cscorelib2.chat.ChatColors;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.GameMode;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.inventory.InventoryClickEvent;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.plugin.Plugin;
+import org.bukkit.scheduler.BukkitRunnable;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The {@link PortableCharger} is an item that opens
+ * a portable charging GUI that charges any
+ * {@link Rechargeable} item.
+ *
+ * @author FluffyBear
+ *
+ */
+public class PortableCharger extends SimpleSlimefunItem<ItemUseHandler> implements Listener, Rechargeable {
+
+    private final int[] border = { 5, 6, 7, 14, 16, 23, 24, 25 };
+    private final int powerSlot = 11;
+    private final int chargeSlot = 15;
+    private final int chargeSpeed = 50;
+    private final Plugin plugin = LiteXpansion.getInstance();
+
+    public PortableCharger() {
+        super(Items.LITEXPANSION, Items.PORTABLE_CHARGER, RecipeType.ENHANCED_CRAFTING_TABLE, new ItemStack[] {
+            Items.COPPER_CABLE, Items.REFINED_IRON, Items.COPPER_CABLE,
```

Comment:
copper on the outside ouch that electrostatic charge

Reply:
change recipe or add a chance for user to get electrocuted

Reply:
Change recipe